applications/ecallmgr/src/ecallmgr_fs_route.erl

```diff
@@ -282,7 +282,7 @@ search_for_route(Section, Node, FetchId, CallId, Props) ->
             ToSet = [{<<"Context">>, hunt_context(Props)}] ++ remote_sdp(Props),
             maybe_wait_for_authz(Section, Node, FetchId, CallId
                                  ,wh_json:set_values(ToSet, JObj)
-                                )
+                                 , Props)
```

Comment:
@icehess why are you passing the large Props down ? you can call ecallmgr_uitil:custom_sip_headers(Props) and add it to the ToSet var above, then extract in ecallmgr_fs_xml.